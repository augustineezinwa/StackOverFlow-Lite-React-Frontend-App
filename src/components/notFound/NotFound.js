import React, { Component, Fragment } from 'react';

class NotFound extends Component {
  constructor(props) {
    super(props);
    this.state = {};
  }

  render() {
    return (
      <Fragment>
        <div className="container image-background" style={{ margin: '0 auto' }}>

          <div className="row">

            <div className="col mt-10">
              <h1 style={{ color: '#FFFFFF', textAlign: 'center' }}>Welcome to StackOverFlow-Lite</h1>
            </div>
          </div>
        </div>

        <div className="container dashboardfooter " id="dashBoardTitle"><h3>Trending Questions</h3></div>

        <div id="questionsDisplay">
          <div className="container">
            <div className="row no-questions">

              <div className="alignCardWidth">
                <div className="card">
                  <div className="container">
                    <div className="row mt-4 pd-1">
                      <div className="col-2">
                        <div className="symbol-display">
                          <div className="alignSymbol">!</div>
                        </div>

                      </div>
                      <div className="col-5">
                        <div className="question">We cant find the page you are looking for </div>

                      </div>
                    </div>

                    <div className="col" style={{ textAlign: 'right' }}>
                      <span />
                      <span />
                      <a href="/">
                        <button type="answer">Refresh</button>
                      </a>

                    </div>

                  </div>
                </div>
              </div>

            </div>
          </div>

        </div>

      </Fragment>
    );
  }
}

export default NotFound;
