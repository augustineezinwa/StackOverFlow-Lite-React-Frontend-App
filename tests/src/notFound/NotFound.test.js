import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../../../src/components/notFound/NotFound';

test('NotFound tests', () => {
  const notFoundComponent = shallow(<NotFound />);
  expect(notFoundComponent).toMatchSnapshot();
});
