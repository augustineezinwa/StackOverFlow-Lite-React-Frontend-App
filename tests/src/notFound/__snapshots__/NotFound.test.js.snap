// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound tests 1`] = `
<Fragment>
  <div
    className="container image-background"
    style={
      Object {
        "margin": "0 auto",
      }
    }
  >
    <div
      className="row"
    >
      <div
        className="col mt-10"
      >
        <h1
          style={
            Object {
              "color": "#FFFFFF",
              "textAlign": "center",
            }
          }
        >
          Welcome to StackOverFlow-Lite
        </h1>
      </div>
    </div>
  </div>
  <div
    className="container dashboardfooter "
    id="dashBoardTitle"
  >
    <h3>
      Trending Questions
    </h3>
  </div>
  <div
    id="questionsDisplay"
  >
    <div
      className="container"
    >
      <div
        className="row no-questions"
      >
        <div
          className="alignCardWidth"
        >
          <div
            className="card"
          >
            <div
              className="container"
            >
              <div
                className="row mt-4 pd-1"
              >
                <div
                  className="col-2"
                >
                  <div
                    className="symbol-display"
                  >
                    <div
                      className="alignSymbol"
                    >
                      !
                    </div>
                  </div>
                </div>
                <div
                  className="col-5"
                >
                  <div
                    className="question"
                  >
                    We cant find the page you are looking for 
                  </div>
                </div>
              </div>
              <div
                className="col"
                style={
                  Object {
                    "textAlign": "right",
                  }
                }
              >
                <span />
                <span />
                <a
                  href="/"
                >
                  <button
                    type="answer"
                  >
                    Refresh
                  </button>
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</Fragment>
`;
